src/iOS.Core/Services/CertificateService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Threading.Tasks;
+using Bit.Core.Abstractions;
+using Bit.Core.Models;
+
+namespace Bit.iOS.Core.Services
+{
+    public class CertificateService : ICertificateService
```

Comment:
I incorrectly assumed this had already been implemented on iOS, and we require platform parity (where possible) for community PRs.  Are you planning to add this?

Reply:
@mpbw2 I would be willing to add for iOS but i've no dev/test envrionment to implement for iOS

Reply:
> @mpbw2 I would be willing to add for iOS but i've no dev/test envrionment to implement for iOS

@oguzhane I'd be happy to send you a cheap/used iPhone from Swappa if you don't have one and you're willing/able to put the time in to iOS support

If so, you can reach me at:

github@<mygithubusername>.com

Either way, I appreciate the work you've done on this, if that's not obvious

I only need to know where to ship it; optionally, I may be able to find a gift card to send via e-mail, and you can handle ordering and shipping, if you have any privacy concerns